src/app/shell/About.tsx

```diff
@@ -126,7 +126,8 @@ export default function About() {
           <div>
             <h2>
               <ExternalLink href={openCollectiveLinkDirect}>
-                <AppIcon icon={heartIcon} /> {t('Views.Support.Support')}
+                <AppIcon icon={heartIcon} />
+                {t('Views.Support.Support')}
```

Comment:
Unintended changes?

Reply:
Yeah this was a style/semantic tweak I failed to omit. 

There are space characters inside the anchor links — this looks like a hack to add spacing between the icon/text. I moved the space to css and styled the links a little clearer

Left: this commit. Right: existing about links.

![image](https://user-images.githubusercontent.com/1396158/236189980-eabbac76-30af-40af-b5ec-fafec9b6a57a.png)

Happy to remove this or could keep it in since it's so minor — whatever you prefer?